### PR TITLE
CDB-34 Added Docker files and scripts to make the management API work…

### PR DESCRIPTION
**Pull request by sbtourist**
… with Astra-flavoured OSS distributions.

### CODE REVIEW

File: astra-4.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,105 @@
+FROM adoptopenjdk:8-jre-hotspot-bionic
```
Comment (sbtourist) [inline astra-4.0/Dockerfile:1]: @stef1927 FYI I've amended this back to a jre image, same as the upstream OSS one (while I previously switched to a jdk image for a test).

**Comment by sbtourist** [comment]
Thanks @stef1927. @tjake and/or @jimdickinson, any interest in reviewing this?

**Comment by tjake** [comment]
Do you want to publish this to docker as well if tagged?

**Comment by tjake** [comment]
https://github.com/datastax/management-api-for-apache-cassandra/blob/master/.github/workflows/docker-release.yaml

**Comment by sbtourist** [comment]
> Do you want to publish this to docker as well if tagged?

@tjake It is currently tagged as 4.0, but such tag would be continuously updated as we develop it. I'm not sure that's a good practice. I'm also wondering if it makes sense to release it as "4.0" given we're soon going to diverge significantly from OSS 4.0. I'd say not to publish it until we figure things out.

File: scripts/docker-entrypoint.sh [hunk 2]
```diff
@@ -70,36 +70,66 @@ if [ "$1" = 'mgmtapi' ]; then
...
-	# for rackdc in dc rack; do
-	# 	var="CASSANDRA_${rackdc^^}"
-	# 	val="${!var}"
-	# 	if [ "$val" ]; then
-	# 		_sed-in-place "$CASSANDRA_CONF/cassandra-rackdc.properties" \
-	# 			-r 's/^('"$rackdc"'=).*/\1 '"$val"'/'
-	# 	fi
-	# done
+    # Set this if you want to ignore default env variables, i.e. when running inside an operator
+    if [ $IGNORE_DEFAULTS ]; then
```
Comment (jimdickinson) [inline scripts/docker-entrypoint.sh:74]: this needs to be the default way this image works for compatibility with published operator versions

**Pull request merged**